src/index.js

```diff
@@ -108,16 +108,17 @@ export default declare(({
         opts.SVG_DEFAULT_PROPS_CODE = t.objectExpression(defaultProps);
       }
 
+      file.get('ensureReact')();
+      file.set('ensureReact', () => {});
+
       if (opts.SVG_DEFAULT_PROPS_CODE) {
         const svgReplacement = buildSvg(opts);
-        path.replaceWithMultiple(svgReplacement);
-      } else {
-        const svgReplacement = buildSvg(opts);
-        path.replaceWith(svgReplacement);
+        return path.replaceWithMultiple(svgReplacement)[0];
```

Comment:
the return value here isn't used, so let's not use `return` here in this way. this should be an if/else instead, as it is in master.